sphinx/source/docs/user_guide.rst

```diff
@@ -74,7 +74,8 @@ The examples in the user guide are written to be as minimal as possible,
 while illustrating how to accomplish a single task within Bokeh. With a
 handful of exceptions, no outside libraries such as NumPy, Pandas, or
 Blaze are required to run the examples as written. However, Bokeh works
-perfectly well with almost any array or table-like data structure.
+perfectly well with NumPy, Pandas, Blaze, or almost any array or table-like 
```

Comment:
I know it was already there but looking at this now I think I'd elide the "perfectly" as superfluous